spec/commands/update_spec.rb

```diff
@@ -34,14 +35,9 @@
   end
 
   describe "--quiet argument" do
-    it "shows UI messages without --quiet argument" do
-      bundle "update"
-      expect(out).to include("Fetching source")
-    end
-
-    it "does not show UI messages with --quiet argument" do
+    it "hides UI messages" do
```

Comment:
I think this was just part of the merge, and on master I deleted the test for showing output, because the output is tested millions of other places.